upload.go

```diff
@@ -0,0 +1,93 @@
+package malwarebazaar
+
+import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"time"
+)
+
+// UploadFile takes the file contents passed any uploads it to malware bazaar with the given fields set. This function doesn't have the ability to set comments or context for the sample
+func UploadFile(ctx context.Context, filename string, contents []byte, anonymous bool, tags []string, deliveryMethod string, apikey string) ([]byte, error) {
+	httpCli := http.Client{
+		Timeout: 15 * time.Second,
+	}
```

Comment:
You should accept a http client, and if nil use the `http.DefaultClient`.  Not a huge deal.